### PR TITLE
Added Rust Installer

### PR DESCRIPTION
**Pull request by tyroruyk**

### CODE REVIEW

File: installer.sh [hunk 1]
```diff
@@ -26,7 +26,13 @@ do_installation() {
 	if cargo build --release; then
     echo "Build complete"
   else
-    echo "Building failed. Exiting"
+    while true; do
+      read -r -p "Cargo was not found! Want to install it? [Y/N] " yn
+      case $yn in
+        [Yy]* ) curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh; break;;
```
Comment (mdgaziur) [inline installer.sh]: Uhh, it'll just install cargo, but not compile Findex

Comment (mdgaziur) [reply installer.sh]: Also, build might fail even if rust is installed(linking issues and other stuff)

**Comment by mdgaziur** [comment]
Check the review

File: installer.sh [hunk 2]
```diff
@@ -26,7 +26,13 @@ do_installation() {
 	if cargo build --release; then
     echo "Build complete"
   else
```
Comment (mdgaziur) [inline installer.sh:28]: The build may fail cuz of linking issues and other stuff. So, assuming rust is installed is a bad idea. I think it'd be better to just show a message to ask the user to install Rust themselves.

Comment (tyroruyk) [reply installer.sh:28]: I think automation is better

Comment (mdgaziur) [reply installer.sh:28]: Leaving the installation to user is better

**Comment by mdgaziur** [comment]
Installer doesn't do what it's supposed to do so closing the PR

**Pull request closed without merging**